pages/index.tsx

```diff
@@ -40,9 +40,20 @@ const Home = () => {
               color="blue"
               icon={<ArrowRightIcon className="w-4 h-4" />}
               href="/apply"
+              fullWidth={true}
+            />
+            <Button
+              title="Om opptak"
+              color="white"
+              href="/about"
+              fullWidth={true}
+            />
+            <Button
+              title="Komiteer"
+              color="white"
+              href="/committees"
+              fullWidth={true}
```

Comment:
Hvis en boolean prop skal være true, kan man forkorte det sånn:

```suggestion
              fullWidth
            />
            <Button
              title="Om opptak"
              color="white"
              href="/about"
              fullWidth
            />
            <Button
              title="Komiteer"
              color="white"
              href="/committees"
              fullWidth
```